testbenches/simlibs.mk

```diff
@@ -185,6 +185,39 @@ $(TESTBENCH_PATH)/synopsys_sim.setup: $(TESTBENCH_PATH)/gen_simlibs.tcl
 	cd $(TESTBENCH_PATH) && vivado -mode batch -source $<
 	echo "cosim_wrapper : $(TESTBENCH_PATH)/vcs_simlibs/cosim_wrapper/64" >> $@
 
+# To reduce the number of warnings to ones we control, we dynamically genrate msg_config file
+$(TESTBENCH_PATH)/msg_config:
+	$(eval XILINX_VIVADO=$(realpath $(XILINX_VIVADO)))
+
+	@echo "{+warn=noTFIPC;  +file=$(HDK_SHELL_DESIGN_DIR)/sh_ddr/sim/sh_ddr.sv;}" >> $@
```

Comment:
I mean, `+warn=none <filename>`?

Reply:
I also just discovered that. It's in the branch I created for Tommy's idea